### PR TITLE
Update ASPNETCORE Dependencies to 6.0.5

### PR DESCRIPTION
**Pull request by TanayParikh**
Not merging https://github.com/dotnet/maui/pull/6081 as we don't want to merge 6.0.6 changes yet.

### CODE REVIEW

**Review by jonathanpeppers — APPROVED** [review]
The changes look OK to me.

I think this is up to @Redth if our release branch is good, and we can put other things like this in main now.

File: NuGet.config [hunk 1]
```diff
@@ -4,17 +4,14 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-aspnetcore -->
+    <add key="darc-pub-dotnet-aspnetcore-3083783" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-30837836/nuget/v3/index.json" />
```
Comment (rmarinho) [inline NuGet.config:7]: Do we still need this feed? Users should be able to get all these packages from nuget.org

Comment (TanayParikh) [reply NuGet.config:7]:
```suggestion
```

**Pull request merged**